docs/_static/js/docversion.js

```diff
@@ -18,16 +18,23 @@
  */
 
 /* Set the version of the website */
-function setVersion(){
-        let doc = window.location.pathname.match(/^\/(api\/.*)$/) || window.location.pathname.match(/^\/versions\/[^*]+\/(api\/.*)$/);
+function setVersion(anchor){
+        if (arguments.length==0) {
+            anchor = window.location.hash
+        };
+        console.log('anchor: ', anchor, arguments.length, window.location.hash, window.location.search);
```

Comment:
I suggest to remove this debugging statement from production code.

Reply:
Thanks, I removed all of the debug statements.